### PR TITLE
Add smt configuration test

### PR DESCRIPTION
**Pull request by dzedro**
test installation and upgrade with smt pattern, basic configuration via smt-wizard and validation with smt-repos smt-sync return value

installation
http://10.100.98.90/tests/1852/modules/smt/steps/1
migration
http://10.100.98.90/tests/1853/modules/smt/steps/1

### CODE REVIEW

**Comment by okurz** [comment]
LGTM

File: products/sle/main.pm [hunk 1]
```diff
@@ -775,6 +775,9 @@ sub load_extra_test () {
 sub load_x11tests() {
     return unless (!get_var("INSTALLONLY") && get_var("DESKTOP") !~ /textmode|minimalx/ && !get_var("DUALBOOT") && !get_var("RESCUECD") && !get_var("HACLUSTER"));
 
+    if (get_var("PATTERNS", '') || get_var('HDD_1', '') =~ /smt/) {
```
Comment (sysrich) [inline products/sle/main.pm]: HDD_1 includes smt? What is that bit for?

Comment (dzedro) [inline products/sle/main.pm]: for upgrade, HDD_1=SLES-12-SP1-GM-gnome-x86_64-smt.img

Comment (dzedro) [inline products/sle/main.pm]: I just used /smt/ It matches also `PATTERNS` and I could not imagine collision with another HDD_1

File: products/sle/main.pm [hunk 2]
```diff
@@ -70,6 +70,10 @@ sub is_sles4sap () {
     return get_var('FLAVOR', '') =~ /SAP/;
 }
 
+sub is_smt () {
+    return (get_var("PATTERNS", '') || get_var('HDD_1', '')) =~ /smt/;
```
Comment (dzedro) [inline products/sle/main.pm]: @coolo updated, condition without brackets did work, but it was wrong ... any PATTERN would match

**Pull request merged**